_includes/swagger.html

```diff
@@ -1,41 +1,76 @@
-  <link href='{{ site.baseurl }}/swagger-ui/css/typography.css' media='screen' rel='stylesheet' type='text/css'/>
-  <link href='{{ site.baseurl }}/swagger-ui/css/screen.css' media='screen' rel='stylesheet' type='text/css'/>
-  <link href='{{ site.baseurl }}/swagger-ui/css/print.css' media='print' rel='stylesheet' type='text/css'/>
-  
-  <script src='{{ site.baseurl }}/swagger-ui/lib/jquery.slideto.min.js' type='text/javascript'></script>
-  <script src='{{ site.baseurl }}/swagger-ui/lib/jquery.wiggle.min.js' type='text/javascript'></script>
-  <script src='{{ site.baseurl }}/swagger-ui/lib/jquery.ba-bbq.min.js' type='text/javascript'></script>
-  <script src='{{ site.baseurl }}/swagger-ui/lib/handlebars-2.0.0.js' type='text/javascript'></script>
-  <script src='{{ site.baseurl }}/swagger-ui/lib/underscore-min.js' type='text/javascript'></script>
-  <script src='{{ site.baseurl }}/swagger-ui/lib/backbone-min.js' type='text/javascript'></script>
-  <script src='{{ site.baseurl }}/swagger-ui/swagger-ui.js' type='text/javascript'></script>
-  <script src='{{ site.baseurl }}/swagger-ui/lib/highlight.7.3.pack.js' type='text/javascript'></script>
-  <script src='{{ site.baseurl }}/swagger-ui/lib/marked.js' type='text/javascript'></script>
-  
-  
+  <link href='http://petstore.swagger.io/css/typography.css' media='screen' rel='stylesheet' type='text/css'/>
+
+  <link href='http://petstore.swagger.io/css/screen.css' media='screen' rel='stylesheet' type='text/css'/>
+  <link href='http://petstore.swagger.io/css/reset.css' media='print' rel='stylesheet' type='text/css'/>
+  <link href='http://petstore.swagger.io/css/print.css' media='print' rel='stylesheet' type='text/css'/>
+  <script src='http://petstore.swagger.io/lib/jquery-1.8.0.min.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/jquery.slideto.min.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/jquery.wiggle.min.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/jquery.ba-bbq.min.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/handlebars-2.0.0.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/underscore-min.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/backbone-min.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/swagger-ui.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/highlight.7.3.pack.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/marked.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/swagger-oauth.js' type='text/javascript'></script>
```

Comment:
You still need to add these files to the project and serve them as part of the app. Don't load them from a 3rd party unless that URL intended to act as a CDN (and even then, do so with caution). Ideally, you shouldn't need an internet connection to run the app locally.